README.md

```diff
@@ -77,13 +85,13 @@ http://172.20.0.6/login.pl allows to log into the company.
 
 Three commands exist to run tests:
 
-* `make test`  
+* `make test`
```

Comment:
The two spaces at the end force a line break, which was intentional. Does it need to go?

Reply:
Trailing spaces are always removed. A trailing `\` does the same